### ISSUE TITLE
dorsal intercalated segment instability deformity

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10728]
**Preferred term label:**
dorsal intercalated segment instability deformity

**Synonyms**

**Definition (free text, please give PubMed ID)**
Lateral wrist radiograph showing extension of the lunate with a scapholunate angle >70°

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Scapholunate (SL) ligament injury
Perilunate dislocation (PLD)
Carpal instability nondissociative (CIND)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001393
- label: Dorsal intercalated segment instability
- parents: [Abnormal wrist physiology (HP:0025799)](https://hpo.jax.org/app/browse/term/HP:0025799)
- pmid: PMID:23790423
- def: A nondissociative form of carpal instability, in which the lunate bone angulates to the dorsal side of the hand.
- comment: Carpal instability can be defined as the lack of ligamentous and bony support to maintain a wrist stable to external forces.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10728
- synonyms:
